source/whats-new.txt

```diff
@@ -32,6 +32,10 @@ Learn what's new in:
 Upcoming Breaking Changes
 -------------------------
 
+- The ``getStats()`` and ``isCapped()`` instance methods of the
+  ``DBCollection`` class are deprecated. The corresponding server commands are
+  deprecated in MongoDB v6.2 and later. To learn how to retrieve this
```

Comment:
S: improve clarity 
```suggestion
  deprecated in MongoDB Server version 6.2 and later. To learn how to retrieve this
```

Reply:
same for the sentence in what's new

Reply:
Thanks for raising this. A couple years ago, our team discussed representing MongoDB server and versions in this format, but it hasn't been closely followed. This is currently consistent with the other Upcoming Breaking Changes item and most of the "What's New" entries in the Java docs, so I think it might be better to leave for now.

I purposely haven't commented on instances in which you've represented it as the suggested version since I don't think it's necessary or high value to propagate to all driver documentation, but rather to stay consistent within a docs set.